packages/screens/FeedPostView/components/FeedPostArticleMarkdownView.tsx

```diff
@@ -0,0 +1,397 @@
+import { SocialCardFooter } from "@/components/socialFeed/SocialCard/SocialCardFooter";
+import { SocialCardHeader } from "@/components/socialFeed/SocialCard/SocialCardHeader";
+import { SocialCardWrapper } from "@/components/socialFeed/SocialCard/SocialCardWrapper";
+import { SpacerColumn, SpacerRow } from "@/components/spacer";
+import { useFetchComments } from "@/hooks/feed/useFetchComments";
+import { useAppNavigation } from "@/hooks/navigation/useAppNavigation";
+import { useIsMobile } from "@/hooks/useIsMobile";
+import { useMaxResolution } from "@/hooks/useMaxResolution";
+import { useNSUserInfo } from "@/hooks/useNSUserInfo";
+import { parseUserId } from "@/networks";
+import {
+  renderHtmlTagStyles,
+  articleMd as md,
+  renderHtmlDomVisitors,
+} from "@/utils/feed/markdown";
+import { zodTryParseJSON } from "@/utils/sanitize";
+import {
+  ARTICLE_MAX_WIDTH,
+  DEFAULT_USERNAME,
+  LINES_HORIZONTAL_SPACE,
+  SOCIAl_CARD_BORDER_RADIUS,
+} from "@/utils/social-feed";
+import { neutral33 } from "@/utils/style/colors";
+import {
+  layout,
+  RESPONSIVE_BREAKPOINT_S,
+  screenContentMaxWidth,
+} from "@/utils/style/layout";
+import { tinyAddress } from "@/utils/text";
+import {
+  OnPressReplyType,
+  PostCategory,
+  ReplyToType,
+  ZodSocialFeedArticleMarkdownMetadata,
+} from "@/utils/types/feed";
+
+const contentPaddingHorizontal = layout.spacing_x2;
+
+export const FeedPostArticleMarkdownView: FC<{
+  post: Post;
+  refetchPost: () => Promise<any>;
+  isLoadingPost?: boolean;
+}> = ({ post, refetchPost, isLoadingPost }) => {
+  const navigation = useAppNavigation();
+  const { width: windowWidth } = useWindowDimensions();
+  const { width } = useMaxResolution();
+  const isMobile = useIsMobile();
+  const [parentOffsetValue, setParentOffsetValue] = useState(0);
+
+  const authorId = post?.authorId;
+  const authorNSInfo = useNSUserInfo(authorId);
+  const [, authorAddress] = parseUserId(post?.authorId);
+  const username = authorNSInfo?.metadata?.tokenId || authorAddress;
+
+  const [localPost, setLocalPost] = useState(post);
+  const feedInputRef = useRef<NewsFeedInputHandle>(null);
+  const [replyTo, setReplyTo] = useState<ReplyToType>();
+  const aref = useAnimatedRef<Animated.ScrollView>();
+  const [flatListContentOffsetY, setFlatListContentOffsetY] = useState(0);
+  const [articleOffsetY, setArticleOffsetY] = useState(0);
+  const [articleWidth, setArticleWidth] = useState(0);
+  const [renderHtmlWidth, setRenderHtmlWidth] = useState(0);
+  const isGoingUp = useSharedValue(false);
+  const isLoadingSharedValue = useSharedValue(true);
+  const [isCreateModalVisible, setCreateModalVisible] = useState(false);
+  const {
+    data: comments,
+    refetch: refetchComments,
+    hasNextPage,
+    fetchNextPage,
+    isLoading: isLoadingComments,
+  } = useFetchComments({
+    parentId: post.id,
+    totalCount: post.subPostLength,
+    enabled: true,
+  });
+  const isNextPageAvailable = useSharedValue(hasNextPage);
```

Comment:
Why we have to use useSharedValue here ? 
I think useRef should be enough in this case